lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -332,6 +333,21 @@ var WebMapServiceCatalogItem = function(terria) {
     "_allLayersInRawMetadata"
   ]);
 
+  knockout.defineProperty(this, "_colorRange", {
+    get: function() {
+      return this.colorScaleMaximum + "  " + this.colorScaleMinimum;
```

Comment:
How about putting them in an array rather than concatenating strings?

```suggestion
      return [this.colorScaleMaximum, this.colorScaleMinimum];
```

Reply:
What would be the benefit of putting them in an array than concat into strings?

Reply:
Oh sorry it's pretty much just a nitpick. But the (small) advantages are:
* it's slightly faster (not that anyone cares here)
* the value is actually usable. We could theoretically get that property and use the elements of the array for something. The string is not likely to be useful to anyone. You'd have to parse it to get the individual values, and even if you just wanted to display them together it's unlikely that the formatting used here would be appropriate for that.